RecoEgamma/Configuration/python/RecoEgamma_EventContent_cff.py

```diff
@@ -188,3 +188,10 @@
                                                                            'keep recoPhotons_islandPhotons_*_*'
                                                                            ])
                     )
+
+from Configuration.Eras.Modifier_ppRef_2017_cff import ppRef_2017
+for ec in [RecoEgammaAOD.outputCommands, RecoEgammaRECO.outputCommands, RecoEgammaFEVT.outputCommands]:
+    ppRef_2017.toModify( ec, func=lambda outputCommands: outputCommands.extend(['keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppGED_*_*',
+                                                                                'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerpp_*_*'
```

Comment:
is there an obvious need to be surgical here and not simply follow the same mods as for pA_2016 above?

Reply:
same as the previous comment. islandPhotons are not added by this. So I made a separate block. If it works, I move it to the block of pA_2016.